.github/workflows/integration-wf.yml

```diff
@@ -0,0 +1,85 @@
+name: Integration Test Suite WorkFlow
+
+on: 
+  push:
+    branches:
+      - integration_test
+  pull_request:
+    branches:
+      - master
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the current repo
+        uses: actions/checkout@v1
+      - name: Install Tool
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-linux-amd64
```

Comment:
I don't know if we can parameterize this so that in the future it is easier to update the K8S version. If it is too much effort, we can do in a later PR

Reply:
i can check ip it can be done dynamically, if possible or other ways to achieve the same.